dockernet/src/init_chain.sh

```diff
@@ -362,7 +362,7 @@ for (( i=2; i <= $NUM_NODES; i++ )); do
     genesis_json="${STATE}/${node_name}/config/genesis.json"
 
     # add the main node as a persistent peer
-    sed -i -E "s|persistent_peers = .*|persistent_peers = \"${MAIN_NODE_ID}\"|g" $config_toml
+    sed -i -E "s|\bpersistent_peers = .*|persistent_peers = \"${MAIN_NODE_ID}\"|g" $config_toml
```

Comment:
what does the `\b` do / is it the same as doing `^persistent_peers`?

Reply:
Not exactly the same but in this case either would work. \b is word boundary so it can't be part of another word.  Applies to new lines, spaces, etc.  The carat only applies to new lines.  In this case the one we want to update is on a new line.  I'll update it to carat for simplicity